safe/gui/tools/wizard/step_kw57_extra_keywords.py

```diff
@@ -77,6 +80,26 @@ def set_widgets(self):
             volcano_name_checkbox = QCheckBox(tr('Volcano Name'))
             volcano_name_line_edit = QLineEdit()
 
+            # Volcano longitude
+            volcano_longitude_checkbox = QCheckBox(
+                tr('Volcano longitude'))
```

Comment:
For all these fields (and previous ones), why not using the extra keyword title and description as tooltip based on definitions?

Reply:
ah yes, I forget to do it. I only add the tooltip. Thanks for the review.

Reply:
from definitions?

Reply:
Yes, this one:
```
                widgets[0].setText(widgets[2]['description'])
                widgets[0].stateChanged.connect(widgets[1].setEnabled)
                widgets[0].setChecked(True)
                widgets[0].setToolTip(widgets[2]['description'])
                widgets[1].setToolTip(widgets[2]['description'])
```
widgets[2] is a list of [checbox, input_widget, extra_keywords_definition]